students/Osiddiquee/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,9 @@
+for n in range (1,100):
+    if n%15 == 0:
```

Comment:
Nice! I like the clever least common multiple idea.

A couple of notes:
- range(1, 100) returns to 99, and I think the assignment wanted to 100. The second argument of range should be one higher than you want.
- Here's what the spacing should be, stylistically, according to PEP8:
```
for n in range(1, 101):
    if n % 15 == 0:
#etc. etc.
```